erpnext/stock/doctype/purchase_receipt/purchase_receipt.py

```diff
@@ -217,6 +217,7 @@ def get_current_stock(self):
 				d.current_stock = bin and flt(bin[0]['actual_qty']) or 0
 
 	def update_batch_with_supplier(self):
+		# update batch doc with supplier if batch doc don't have supplier
```

Comment:
This comment is pretty useless, it does nothing that the function header does not already explain.